build.gradle.kts

```diff
@@ -12,15 +12,38 @@ plugins {
     id("org.liquibase.gradle") version Versions.liquibaseGradlePlugin
 }
 
+val profile = if(properties["profile"] == null) "dev" else properties["profile"]
+val propertyFileLines = File("save-backend/src/main/resources/application-$profile.properties").readLines()
+val secretsLines = File("secrets").readLines() // TODO: specify path to file
+
+val databaseUrl = getCredential(propertyFileLines, "database.url")
```

Comment:
Why not the same property as spring datasource? Will this `database.url` be reused there?

Reply:
Made to spring.datasource